src/graphql/mutations.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from '@apollo/client';
+
+export const LOGIN_MUTATION = gql`
+  mutation Login($data: LoginInput!) {
+    login(data: $data) {
+      token
+    }
+  }
+`;
```

Comment:
[EOF](https://en.wikipedia.org/wiki/End-of-file#:~:text=In%20computing%2C%20end%2Dof%2D,called%20a%20file%20or%20stream.)